index.html

```diff
@@ -0,0 +1,84 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Tribute Page</title>
+  <link rel="stylesheet" href="./styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+</head>
+
+<body>
+ <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+ <script src="js/index.js"></script>
+  <main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <p class="text-center"><em>The man who saved a billion lives</em></p>
+    <figure id="img-div"><img id="image"
+        src="https://images.ctfassets.net/hrltx12pl8hq/7yQR5uJhwEkRfjwMFJ7bUK/dc52a0913e8ff8b5c276177890eb0129/offset_comp_772626-opt.jpg?fit=fill&w=800&h=300">
+      <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to
+        increase wheat yields - part of his life-long war on hunger. </figcaption>
+    </figure>
+    <p>Here's a time line of Dr. Borlaug's life:
+    </p>
```

Comment:
- [ ] I think it's better to include an explanation, a "why", for each requested change. For example:

This is a well-explained request: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

This request lacks a "why": Why don't you use an article tag here instead of a div?